packages/dd-trace/src/config.js

```diff
@@ -193,6 +198,7 @@ class Config {
     this.appsec = {
       enabled: isTrue(DD_APPSEC_ENABLED)
     }
+    this.isInCI = isTrue(DD_CI_VISIBILITY_ENABLED)
```

Comment:
Why a different name than the configuration option? This would make the source of the config more difficult to find.